websockets/websockets.go

```diff
@@ -223,12 +230,11 @@ func (w *webSocket) establishConnection() {
 		// EnableCompression: enableCompression,
 		// Jar:               jar,
 	}
-	// TODO figure out cookie jar given the specification
-	header := make(http.Header)
-	header.Set("User-Agent", state.Options.UserAgent.String)
+
+	params.headers.Set("User-Agent", state.Options.UserAgent.String)
```

Comment:
nitpick:
This might actually be a good idea to be done on the event loop (so before calling this function in a goroutine). 

As  `state.Options` isn't immutable and can be changed  ... but it is more likely that it won't either way.

Reply:
@mstoykov Does 64f5929 resolve it?